video/out/vo_gpu_next.c

```diff
@@ -843,7 +845,7 @@ static void apply_target_options(struct priv *p, struct pl_frame *target,
         tbits->sample_depth = dither_depth;
     }
 
-    if (opts->icc_opts->icc_use_luma && pl_color_transfer_is_hdr(target->color.transfer)) {
+    if (opts->icc_opts->icc_use_luma && use_max_luma) {
```

Comment:
the condition for ICC is now pointless because the whole point is avoiding to set `target-peak` and leave it to being detected from ICC

Reply:
Here `target-peak` is used for HDR tone mapping, the reference white of some HDR videos is 100nit, users can specify the corresponding `target-peak` parameter.